Example/PINRemoteImage Tests/PINRemoteImage_Tests.m

```diff
@@ -234,6 +234,37 @@ - (void)testJPEGDownload
     [self waitForExpectationsWithTimeout:[self timeoutTimeInterval] handler:nil];
 }
 
+- (void)testCallbackQueueDefaultsToMainQueue
+{
+    dispatch_queue_t callbackQueue = self.imageManager.callbackQueue;
+    dispatch_queue_t mainqueue = dispatch_get_main_queue();
+    XCTAssert(callbackQueue == mainqueue);
+    XCTestExpectation *expectation = [self expectationWithDescription:@"Downloading JPEG image"];
+    [self.imageManager downloadImageWithURL:[self JPEGURL]
+                                    options:PINRemoteImageManagerDownloadOptionsNone
+                                 completion:^(PINRemoteImageManagerResult *result)
+     {
+         XCTAssertTrue([[NSThread currentThread] isMainThread]);
+         [expectation fulfill];
+     }];
+    [self waitForExpectationsWithTimeout:[self timeoutTimeInterval] handler:nil];
+}
+
+- (void)testCanUseDifferentCallbackQueue {
+    dispatch_queue_t callbackQueue = dispatch_queue_create("PINRemoteImageManagerCallbackQueue", DISPATCH_QUEUE_CONCURRENT);
```

Comment:
You mean as another test case? Definitely because it defaults to the main queue so its nice to see if we can change this. or am I missing the question?